src/main/java/ai/rev/speechtotext/CustomVocabularyApiInterface.java

```diff
@@ -0,0 +1,23 @@
+package ai.rev.speechtotext;
+
+import ai.rev.speechtotext.models.vocabulary.CustomVocabularyInformation;
+import ai.rev.speechtotext.models.vocabulary.CustomVocabularyOptions;
+import retrofit2.Call;
+import retrofit2.http.Body;
+import retrofit2.http.GET;
+import retrofit2.http.POST;
+import retrofit2.http.Path;
+
+import java.util.List;
+
+public interface CustomVocabularyApiInterface {
```

Comment:
I think this needs documentation like the proceeding ApiInterface: https://github.com/revdotcom/revai-java-sdk/pull/19/files#diff-bc6d97b48980ad0bde766336dd7e5d6bR25